src/styles/pointerEventsAuto/types.ts

```diff
@@ -1,5 +1,5 @@
-import {CSSProperties} from 'react';
+import {ViewStyle} from 'react-native';
 
-type PointerEventsAutoStyles = Partial<Pick<CSSProperties, 'pointerEvents'>>;
+type PointerEventsAutoStyles = Pick<ViewStyle, 'pointerEvents'>;
```

Comment:
what is the difference between `PointerEventsAutoStyles` and `PointerEventsNone`? Are these same?

Reply:
They are different 
`PointerEventsAutoStyles` is a union of possible values which we can pass to `pointerEvents` property
`PointerEventsNone` is just `none` value.